components/navBar/index.tsx

```
@@ -88,6 +88,7 @@ const Nav: React.FC<MyComponentProps> = ({ children }) => {
       permissions: [], // No permissions required
     },
     {
+     
       key: '/leads',
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)